commands/entity-store.mjs

```diff
@@ -12,11 +12,7 @@ let EVENT_INDEX_NAME = "auditbeat-8.12.0-2024.01.18-000001";
 const offset = () => Math.random() * 1000;
 
 const ASSET_CRITICALITY = [
-  "very_important",
-  "not_important",
-  "normal",
-  "important",
-  "unknown",
```

Comment:
Is unknown a value that is used by the data generator? It isnt a valid criticality level

Reply:
Ah yes I see https://github.com/hop-dev/security-documents-generator/blob/30dada4311d0238baf1ee937daf52cfcec5c56f6/commands/entity-store.mjs#L101

Reply:
I think at one point we did have an unknown, probably why that's still here.
We can probably clean this up in the future